english/is-this-a-dream/is-this-a-dream.md

```diff
@@ -1,10 +1,14 @@
 as the sharp pain moves through your nervous system and reaches your brain, the marshmallow walls begin
 to tremble stronger and stronet and brighten up until they blow up around you in a shimmering white light
 that engulfs you...you close your eyes from the strong light and when you open them up...the beautiful 	
-view from your 20th floor in central park NY and the sexy ass of the current babe you conquered last night 
```

Comment:
Didn't change anything, but as a side note, I didn't see the part that was so explicit when I was making changes.